NorthstarDLL/plugins/plugin_abi.h

```diff
@@ -61,6 +57,10 @@ struct SquirrelFunctions
 	sq_getassetType __sq_getasset;
 	sq_getuserdataType __sq_getuserdata;
 	sq_getvectorType __sq_getvector;
+	sq_getthisentityType __sq_getthisentity;
+	sq_getobjectType __sq_getobject;
+
+	sq_stackinfosType __sq_stackinfos;
```

Comment:
This is a breaking change therefore this pr should also increment the plugin version or not get released without plugins v3.

Reply:
Oh wait this struct is accessed by plugins, right? Yeah then it makes no sense to splice out, lol.

I just made the PR on the side while going through the diff ^^